oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/IndexController.java

```diff
@@ -82,6 +106,9 @@ public static class LogicIndicesRegister {
          * The relations of the logic table and the physical table.
          */
         private static final Map<String, String> LOGIC_INDICES_CATALOG = new ConcurrentHashMap<>();
+
+        private static final Map<String, Map<String, ModelColumn>> PHYSICAL_INDICES_COLUMNS = new ConcurrentHashMap<>();
```

Comment:
Please put comments here about what is the nested structure of this Map. This would help reading codes.